### DISCUSSION TITLE
第一次打开PCL2时检测电脑地区/语言并将启动器的语言改为这台电脑的语言

### DISCUSSION THREAD

**Discussion by xudisen** [Meloong-Git/PCL#647] [提问] [answered]
pikaへ　　　　　／|
　　/＼7　　　 ∠＿/
　 /　│　　 ／　／
　│　Z ＿,＜　／　　 /`ヽ
　│　　　　　ヽ　　 /　　〉
　 Y　　　　　`　 /　　/
　ｲ●　､　●　　⊂⊃〈　　/
　()　 へ　　　　|　＼〈
　　>ｰ ､_　 ィ　 │ ／／
　 / へ　　 /　ﾉ＜| ＼＼
　 ヽ_ﾉ　　(_／　 │／／
　　7　　　　　　　|／
　　＞―r￣￣`ｰ―＿6

**Comment by Big-Cake-jpg** [comment, marked as answer]
PCL2 暂不考虑支持多语言，因为这将需要重新设计整个界面。
